frontend/ui/data/css_tweaks.lua

```diff
@@ -85,6 +85,69 @@ h4, h5, h6 { page-break-after: avoid !important; }
                 css = [[* { text-align: justify !important; }]],
             },
         },
+        {
+            title = _("Widows and orphans"),
+            {
+                title = _("About widow and orphan lines"),
+                info_text = _([[
+Widows and orphans are lines at the beginning or end of a paragraph, which are left dangling at the top or bottom of a page, separated from the rest of the paragraph.
+The first line of a paragraph alone at the bottom of a page is called an orphan line.
+The last line of a paragraph alone at the top of a page is called a widow line.
+
+Some people (and publishers) don't like widows and orphans, and can avoid them with CSS rules.
+But to avoid widows and orphans, some lines have to be pushed to the next page to keep company to the othewise widows and orphans, and may leave some blank space at the bottom of the previous page, which some other people may find more disturbing.
+
+The default is to allow widows and orphans.
+These tweaks allow you to change this behaviour, and to override publishers rules.]]),
```

Comment:
Not wrong of course, but more consistent with the other texts.

```suggestion
These tweaks allow you to change this behavior, and to override publisher rules.]]),
```